### DISCUSSION TITLE
Custom Breakpoints (add new size as a props)

### DISCUSSION THREAD

**Discussion by imzirak** [mantinedev/mantine#2729] [Q&A]
Hi,
There are 5 sizes available in Mantine (xs, sm, ..., xl)
Instead, I want to add a new item, like bootstrap, and add the size xxl as a prop size.
How can we handle this, or does Mantine have a solution for configuring it.

**Comment by rtivital** [comment]
Adding custom breakpoints is not supported, you can add them to `theme.other` or use values, for example `theme.fn.smallerThan(700)`
